configs/.env.example

```diff
@@ -1,4 +1,3 @@
-ETH_PRIVATE_KEYS="eeeee6653cdcacc36e3c400ceeeef2aefd59e2642c2f7f298047eeeeeeeeeeee,9643c732204f2a7c9bdb74e2fa08e36d6a4ae8378b983064848b76318fb6507d"
-VIXEOD_KEY=xxx
-CMC_KEY=xxxxxxxxxxxxx
-NODE_WEBSOCKET_URL="wss://mainnet.infura.io/v3/ws/xxxxxxxxxxxxx"
\ No newline at end of file
+ETH_PRIVATE_KEYS="eeeee6653cdcacc36e3c400ceeeef2aefd59e2642c2f7f298047eeeeeeeeeeee,9643c732204f2a7c9bdb74e2fa08e36d6a4ae8378b983064848b76318fb6507d" # required list of private keys separated by `,`   
+$PSR$_KEY=xxxxxxxxxxxxx # API key for getting a specific indexes.json api \(required if you use authenticated API's\)
```

Comment:
This is not very clear. I think instead we can just add somewhere in the docs that in the API file can use any var from the env file which allows to have auth keys in the API urls.
```suggestion
```

Reply:
I don't see any change why marked as resolved?